src/app/zap-templates/zcl/data-model/chip/operational-credentials-cluster.xml

```diff
@@ -24,6 +24,21 @@ limitations under the License.
     <item name="Label" type="OCTET_STRING" length="32"/><!-- TODO: Add Label once CHAR_STRING or OCTET_STRING works -->
   </struct>
 
+  <enum name="NOCResponseStatus" type="ENUM8">
```

Comment:
This is called "NodeOperationalCertStatus" in the spec.  Why the difference in naming?  It makes it harder to match up code to spec....